pkg/appconsts/versioned_consts.go

```diff
@@ -22,6 +24,17 @@ func SubtreeRootThreshold(_ uint64) int {
 	return v1.SubtreeRootThreshold
 }
 
+// GlobalMinGasPrice is used in the processProposal to ensure
+// that all transactions have a gas price greater than or equal to this value.
+func GlobalMinGasPrice(version uint64) (float64, error) {
+	switch version {
+	case v2.Version:
+		return v2.GlobalMinGasPrice, nil
+	default:
+		return 0, fmt.Errorf("global min gas price not defined for version %d", version)
+	}
+}
```

Comment:
[optional] add tests in `versioned_consts_test.go`. The unit test may look crazy simple for now but I expect it may be more helpful down the line when we introduce new app versions.